internal/utils/utils.go

```diff
@@ -263,15 +265,15 @@ func WriteToWorkingDirFile(toFile string, value []byte) error {
 }
 
 func init() {
-	rand.Seed(time.Now().UnixNano())
+	rgen = rand.New(rand.NewSource(time.Now().UnixNano()))
 }
 
 var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")
 
 func GetRandomWord(wordSize int) []byte {
 	b := make([]rune, wordSize)
 	for i := range b {
-		b[i] = letters[rand.Intn(len(letters))] //nolint:gosec
+		b[i] = letters[rgen.Intn(len(letters))] //nolint:gosec
```

Comment:
Since Go 1.20, we can just call `rand.Intn()`. There's no longer a need to manually seed the generator: https://cs.opensource.google/go/go/+/refs/tags/go1.20.6:src/math/rand/rand.go;l=306-313